Example/Backpack/Stories/HorizontalNavigationStory.swift

```diff
@@ -0,0 +1,83 @@
+//
```

Comment:
extraneous comment

Reply:
I think these might be from the project template

Reply:
Or is it that limitation in Xcode?

Reply:
I'm not sure - I just know I try to kill them off whenever I see them